src/main/java/uk/gov/hmcts/ccd/domain/service/casedataaccesscontrol/RoleAssignmentService.java

```diff
@@ -118,8 +118,12 @@ public RoleAssignments getRoleAssignments(String userId) {
     public List<RoleAssignment> getRoleAssignments(String userId, CaseTypeDefinition caseTypeDefinition) {
         final RoleAssignments roleAssignments = this.getRoleAssignments(userId);
         return roleAssignmentsFilteringService
-            .filter(roleAssignments, caseTypeDefinition, Lists.newArrayList(MatcherType.GRANTTYPE))
-            .getFilteredMatchingRoleAssignments();
+            .filter(roleAssignments, caseTypeDefinition,
+                Lists.newArrayList(
+                    MatcherType.GRANTTYPE,
+                    MatcherType.SECURITYCLASSIFICATION
```

Comment:
The initial Role Assignment filtering on Search also shouldn't do the `Authorisations` check - would it be easier/safer to have a method that instead INCLUDES certain matchers and only include `BEGINENDDATE`, `CASETYPE`, `JURISDICTION` and `ROLENAME`?